### PR TITLE
Setting up webpack and react

### PR DESCRIPTION
**Pull request by winstonkim**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+; editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
```
Comment (christianvuerings) [inline .editorconfig]: @winstonkim could we add an extra line at the end of this file?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CalCentral Monitor</title>
```
Comment (christianvuerings) [inline index.html:5]: @winstonkim could you make sure we add a `meta` charset already? (`utf-8`)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/winstonkim/cc-monitor.git"
+  },
+  "keywords": [],
+  "author": "Winston Kim",
+  "license": "ISC",
```
Comment (christianvuerings) [inline package.json]: @winstonkim can we also make this `ECL-2.0`? Just like CalCentral

File: package.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  "bugs": {
+    "url": "https://github.com/winstonkim/cc-monitor/issues"
+  },
+  "homepage": "https://github.com/winstonkim/cc-monitor",
```
Comment (christianvuerings) [inline package.json]: @winstonkim let's use `ets-berkeley-edu` everywhere in this file

**Comment by christianvuerings** [comment]
looks awesome. Let's try to add ESLint in here asap

**Comment by winstonkim** [comment]
@christianvuerings made the changes you requested and added eslint

**Pull request merged**